src/Worlds/DebugWorld/DebugWorld_GravelChain.layer

```diff
@@ -0,0 +1,26 @@
+GenericEntity : "{344090890C63F4CA}Prefabs/Items/Resources/Gravel_bag.et" {
+ coords 128.054 0.001 135.005
+}
+GenericEntity : "{5C5C6EE05EE2FF1A}Prefabs/Items/Equipment/Backpacks/Backpack_ALICE_Medium_assembled.et" {
+ coords 123.012 0.001 135.773
+}
+StaticModelEntity : "{ADBA6500B487DC94}Prefabs/Props/Construction/GravelPile.et" {
+ components {
+  MeshObject "{5272E314F6439B95}" {
+   Object "{352A1A2A8D933FFE}Assets/Props/Construction/GravelPile_01/GravelPile_01_medium.xob"
+  }
+  SimpleSoundComponent "{598A74222DFDE827}" {
+   Filenames +{
+   }
+  }
+  ActionsManagerComponent "{598A74222DFDE823}" {
+   ActionContexts {
+    UserActionContext "{598A74222DFDE81E}" {
+    }
+   }
+  }
+  RplComponent "{598A74222DFDE817}" {
+  }
+ }
```

Comment:
I am not sure why the map saved all of this info. If we just used the prefab it should have just referenced the gravel pile prefab and listed none of the components.